### PR TITLE
Feat: rating history

### PR DESCRIPTION
**Pull request by Setanna**

### CODE REVIEW

File: lib/ratings.dart [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.red,
+        title: const Text("Ratings"),
+        leading: IconButton(
+          icon: Icon(Icons.arrow_back, color: Colors.white),
+          onPressed: () => Navigator.of(context).pop(),
+        ),
+      ),
```
Comment (MathiasDuus) [inline lib/ratings.dart:18]: Det ser ud til at man kan bruge  `const BackButton()`

File: lib/widget/rated_products.dart [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder<List>(
+      future: rating.fetchRatings(),
+      builder: (context, snapshot) {
+        // if fetch favorites has data
+        if (snapshot.hasData) {
+          // return a gridview builder
+          return GridView.builder(
+              // with item count equal to fetch favorites data length
```
Comment (MathiasDuus) [inline lib/widget/rated_products.dart:19]: Er du sikker på du ikke mener fetch ratings?

File: lib/ratings.dart (hunk 1 above)
Comment (Setanna) [reply lib/ratings.dart:18]: Skal nok kigge på det

File: lib/widget/rated_products.dart (hunk 2 above)
Comment (Setanna) [reply lib/widget/rated_products.dart:19]: nej

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Comment by Setanna** [comment]
[My ratings historik](https://trello.com/c/ZCzBnZVt/23-my-ratings-historik)

**Review by MathiasDuus — APPROVED** [review]
Looks good

**Pull request merged**